cloud: add credit docs (#9925)
pingcap/docs: tidb-cloud/tidb-cloud-billing.md

2. Click **Billing**.
3. Click **Bills**. The billing details page is displayed.

## Credits

> You can use credits to pay TiDB cluster fees before the credits become expired. One credit is equivalent to one U.S. dollar.

Reviewer: ```suggestion
TiDB Cloud offers a certain number of credits for Proof of Concept (PoC) users. One credit is equivalent to one U.S. dollar. You can use credits to pay TiDB cluster fees before the credits become expired.

You can use credits to pay TiDB cluster fees before the credits become expired. One credit is equivalent to one U.S. dollar.
```

> 

Author: ```suggestion

> **Tip:**
>
> To apply for a PoC, see [Perform a Proof of Concept (PoC) with TiDB Cloud](/tidb-cloud/tidb-cloud-poc.md)

```

> Different types of credits can be used to pay different fees.

Reviewer: Currently, only the "General" type is supported.

Author: @yuezhanglang PTAL

Author: I have deleted this sentence.

2. Click **Billing**.
3. Click **Bills**. The billing details page is displayed.

## Credits

TiDB Cloud offers a certain number of credits for Proof of Concept (PoC) users. One credit is equivalent to one U.S. dollar. You can use credits to pay TiDB cluster fees before the credits become expired.

> 
> You can use credits to pay TiDB cluster fees before the credits become expired. One credit is equivalent to one U.S. dollar.

Reviewer: ```suggestion
```

2. Click **Billing**.
3. Click **Bills**. The billing details page is displayed.

## Credits

TiDB Cloud offers a certain number of credits for Proof of Concept (PoC) users. One credit is equivalent to one U.S. dollar. You can use credits to pay TiDB cluster fees before the credits become expired.

> **Tip:**
>

> > To apply for a PoC, see [Perform a Proof of Concept (PoC) with TiDB Cloud](/tidb-cloud/tidb-cloud-poc.md)

Reviewer: ```suggestion
> To apply for a PoC, see [Perform a Proof of Concept (PoC) with TiDB Cloud](/tidb-cloud/tidb-cloud-poc.md).
```

> The detailed information of your credits is available on the **Credits** page, including your total credits, available credits, current usage, the status, and the expiry date.

Author: ```suggestion
The detailed information of your credits is available on the **Credits** page, including your total credits, available credits, current usage, and the status.
```

To view this page, perform the following steps:

1. Click the account name in the upper-right corner of the TiDB Cloud console.
2. Click **Billing**.
3. Click **Credits**. The credit details page is displayed.

> **Note:**
>
> - After you set up your payment method, the cluster fees are first deducted from your unused credits, then from your payment method.
> - Credits cannot be used to pay the support plan fees.

> **Warning:**
>
> During a PoC process:
>

> > - if all your credits become expired before you add a payment method, you cannot creat a new cluster. After 3 days, all your created clusters will be recycled. After 7 days, all your backups will be recycled. To resume the process, you can add a payment method.
> > - if all your credits become expired after you add a payment method, your PoC process goes on, and fees are deducted from your payment method.

Reviewer: ```suggestion
> - If all your credits become expired before you add a payment method, you cannot create a new cluster. After 3 days, all your existing clusters will be recycled. After 7 days, all your backups will be recycled. To resume the process, you can add a payment method.
> - If all your credits become expired after you add a payment method, your PoC process goes on, and fees are deducted from your payment method.
```
